code/modules/mob/living/living_say.dm

```diff
@@ -86,7 +86,7 @@ GLOBAL_LIST_EMPTY(channel_to_radio_key)
 			S.message = cultslur(S.message)
 			verb = "slurs"
 
-		if(!IsVocal())
+		if(!IsVocal() || HAS_TRAIT(src, TRAIT_MUTE))
```

Comment:
shouldnt isvocal check for that?

Reply:
IsVocal() is only properly defined for humans